.gitignore

```diff
@@ -18,3 +18,4 @@ Waffle.sln.cache
 Waffle.suo
 *.csproj.user
 .tern-project
+*.iml
```

Comment:
I realize a comment was made to explicitly set impersonate to false.  Jvm does that by default so I'd remove that.  I'll get a better overall look tonight.




Sent from Outlook Mobile






On Wed, Dec 30, 2015 at 4:23 AM -0800, "Sergey Podolsky" <notifications@github.com> wrote:





> @@ -18,3 +18,4 @@ Waffle.sln.cache
>  Waffle.suo
>  *.csproj.user
>  .tern-project
> +*.iml

a module file created by IntelliJ IDEA

---
Reply to this email directly or view it on GitHub:
https://github.com/dblock/waffle/pull/309/files#r48601291